discord_slash/context.py

```diff
@@ -114,6 +114,22 @@ def channel(self) -> typing.Optional[typing.Union[discord.TextChannel, discord.D
         """
         return self.bot.get_channel(self.channel_id)
 
+    @property
+    def voice_client(self):
+        """
+        VoiceClient instance of the command invoke. If the command was invoked in DM, then it is ``None``.
+        If the bot is not connected to any Voice/Stage channels, then it is ``None``.
+
+        """
+        return self.guild.voice_client if self.guild else None
+
+    @property
+    def me(self):
+        """
+        Similar to :attr:`.Guild.me` except that it may also return the bot user in DM context.
+        """
+        return self.guild.me if self.guild is not None else self.bot.user
```

Comment:
`is not` produces a raised `SyntaxWarning` which may not me mutable to some users when logged. Please consider using `!=` instead due to strict checking. The `is` operator checks for identity correlation which should be used in the event where your value can always be assured it has `None` as a value, but otherwise should be using the prior.